### PR TITLE
Refactor kubejobs plugin applying extract method

### PR DESCRIPTION
**Pull request by javanlacerda**
* Split start_application method to smaller methods

### CODE REVIEW

File: broker/plugins/kubejobs/plugin.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def trigger_job(self, data):
     def start_monitoring(self, data):
         monitor.start_monitor(api.monitor_url, self.app_id,
                               data['monitor_plugin'],
-                              data['monitor_info'], 2)
+                              data['monitor_info'], 1)
```
Comment (joseims) [inline broker/plugins/kubejobs/plugin.py]: Magic Number

File: broker/tests/unit/plugin/kubejobs/test_plugin.py [hunk 2]
```diff
@@ -174,11 +191,83 @@ def test_update_monitor_info(self):
                                       queue_size, redis_ip, redis_port)
         self.assertEqual(data.get('monitor_info'), monitor_info_after)
 
-    """
-    Test the Get and Update Application State of
-    the Kubejobs plugin.
-    """
+    def test_start_visualization(self):
```
Comment (joseims) [inline broker/tests/unit/plugin/kubejobs/test_plugin.py]: Please set kj-000001 to a constant and reuse that for better readability

File: broker/tests/unit/plugin/kubejobs/test_plugin.py [hunk 3]
```diff
@@ -236,45 +324,40 @@ def test_wrong_request_body(self):
...
-    """
-
     def setUp(self):
+        """
+        Set up KubeJobsProvider objects
+        """
         self.provider1 = KubeJobsProvider()
         self.provider2 = KubeJobsProvider()
 
     def tearDown(self):
```
Comment (joseims) [inline broker/tests/unit/plugin/kubejobs/test_plugin.py]: Is this really needed if it is not used?

File: broker/tests/unit/plugin/kubejobs/test_plugin.py [hunk 4]
```diff
@@ -236,45 +324,40 @@ def test_wrong_request_body(self):
...
-
     def test_get_description(self):
+        """
+        Test the Get Description of the KubeJobs Provider
+        """
         self.assertEqual(
             self.provider1.get_description(),
             'Plugin that allows utilization of Batch Jobs over a k8s cluster')
         self.assertEqual(
             self.provider1.get_description(),
```
Comment (joseims) [inline broker/tests/unit/plugin/kubejobs/test_plugin.py]: Here is written provider1 twice, shouldn't it be provider 2?

File: broker/tests/unit/plugin/kubejobs/test_plugin.py [hunk 5]
```diff
@@ -285,9 +368,6 @@ def test_to_dict(self):
             to_dict.get("description"),
             'Plugin that allows utilization of Batch Jobs over a k8s cluster')
 
-    """
-    """
-
     def test_execute(self):
```
Comment (joseims) [inline broker/tests/unit/plugin/kubejobs/test_plugin.py]: is this really needed?

File: broker/plugins/kubejobs/plugin.py (hunk 1 above)
Comment (javanlacerda) [reply broker/plugins/kubejobs/plugin.py]: Fixed

File: broker/tests/unit/plugin/kubejobs/test_plugin.py (hunk 2 above)
Comment (javanlacerda) [reply broker/tests/unit/plugin/kubejobs/test_plugin.py]: Done

File: broker/tests/unit/plugin/kubejobs/test_plugin.py (hunk 3 above)
Comment (javanlacerda) [reply broker/tests/unit/plugin/kubejobs/test_plugin.py]: What you're talking about ?

File: broker/tests/unit/plugin/kubejobs/test_plugin.py (hunk 4 above)
Comment (javanlacerda) [reply broker/tests/unit/plugin/kubejobs/test_plugin.py]: Fixed

File: broker/tests/unit/plugin/kubejobs/test_plugin.py (hunk 5 above)
Comment (javanlacerda) [reply broker/tests/unit/plugin/kubejobs/test_plugin.py]: Removed

File: broker/tests/unit/plugin/kubejobs/test_plugin.py (hunk 3 above)
Comment (joseims) [reply broker/tests/unit/plugin/kubejobs/test_plugin.py]: The empty tear_down method, it's not your code, but maybe we could remove the method, what do you think about it?

**Pull request merged**